alchemist-full/build.gradle.kts

```diff
@@ -30,6 +35,98 @@ application {
     mainClass.set("it.unibo.alchemist.Alchemist")
 }
 
+val copyForPackaging by tasks.registering(Copy::class) {
+    val jarFile = "alchemist-full-${rootProject.version}-all.jar"
+    from("${rootProject.projectDir}/build/shadow/$jarFile")
+    into("${rootProject.projectDir}/build/package-input")
+    dependsOn(tasks.shadowJar)
+}
+
+// JPackage
+tasks.jpackage {
```

Comment:
get the task type of JPackage: we can create our own custom jpackage tasks with every generable output